be/src/testing/testing.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, MethodNotAllowedException } from '@nestjs/common';
+import { ItemService } from '../item/item.service';
+import { CreateItemInput } from 'src/item/dto/create-item.input';
+import { CategoryService } from 'src/category/category.service';
+import { CreateCategoryInput } from 'src/category/dto/create-category.input';
+import { CreateNoteInput } from 'src/note/dto/create-note.input';
+import { NoteService } from 'src/note/note.service';
+import { UserService } from 'src/user/service/user.service';
+import { CreateUserDTO } from 'src/user/dto/user.create.dto';
+import { Role } from 'src/user/roles/user.roles';
+import * as faker from 'faker';
+
+@Injectable()
+export class TestingService {
+  constructor(
+    private readonly categoryRepo: CategoryService,
+    private readonly itemRepository: ItemService,
+    private readonly noteRepo: NoteService,
+    private readonly userRepo: UserService,
+  ) {
+    if (process.env.ADD_FAKE_DATA === 'TRUE') this.AddFakeData();
+    console.log('add fake data ', process.env.ADD_FAKE_DATA);
+  }
+
+  AddFakeData() {
+    if (process.env.ADD_FAKE_DATA !== 'TRUE') {
+      throw new MethodNotAllowedException(`ADD_FAKE_DATA not set to TRUE`);
+    }
+    for (let i = 0; i < 3; i++) {
+      const item: CreateItemInput = { regCode: faker.datatype.number() };
+      this.itemRepository.createItem(item);
+
+      const category: CreateCategoryInput = {
+        name: faker.name.findName(),
+        bundelable: faker.datatype.boolean(),
+        parentCategoryId: null,
```

Comment:
maybe create a few child categories as well?

Reply:
done in [a35339](https://github.com/TPT-Loane/TPT-Loane/pull/120/commits/a353392cbcddaaad24948aba9a4e01545c95f900) and [4e2c5b](https://github.com/TPT-Loane/TPT-Loane/pull/120/commits/4e2c5b4cf733f655831cd982cdfd397411a33e1d)